pylib/gyp/generator/msvs.py

```diff
@@ -335,15 +335,15 @@ def _ConfigWindowsTargetPlatformVersion(config_data, version):
             # Find a matching entry in sdk_dir\include.
             expected_sdk_dir = r"%s\include" % sdk_dir
             names = sorted(
-                [
+                (
```

Comment:
Is this a misspell ?

Reply:
Nope.  It is an optimization because tuples (read-only) take less memory than lists (read-write) in Python...
$ `python3`
```
>>> import sys
>>> sys.getsizeof([1, 2, 3])
120
>>> sys.getsizeof((1, 2, 3))
64
```